CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 This is only used for recording changes for major version bumps.
 More minor changes may optionally be recorded here too.
 
+## 60.1.0
```

Comment:
Technically a breaking change because we're removing `get_next_work_day` ... but I don't see that being used anywhere, so it's probably fine.